quixstreams/platforms/quix/config.py

```diff
@@ -359,119 +396,61 @@ def _finalize_create(
                     exceptions[topic["name"]] = topic["lastError"]
                     topics.remove(topic["id"])
         if exceptions:
-            raise QuixCreateTopicFailure(f"Failed to create Quix topics: {exceptions}")
+            raise QuixCreateTopicFailure(f"Quix topic validation failed: {exceptions}")
         if topics:
             raise QuixCreateTopicTimeout(
-                f"Creation succeeded, but waiting for 'Ready' status timed out "
-                f"for topics: {[self.strip_workspace_id_prefix(t) for t in topics]}"
+                f"Waiting for 'Ready' status timed out for Quix "
+                f"topics: {[t['name'] for t in checks if t['id'] in topics]}"
             )
 
-    def create_topics(
+    def confirm_topic_ready_statuses(
```

Comment:
I'd prefer a more descriptive method name. This can block for a while. something like `wait_for_topics_ready_statuses`